src/server/index.ts

```diff
@@ -0,0 +1,15 @@
+import { NestFactory } from '@nestjs/core';
+import { AppModule } from './app.module';
+
+const { SERVER_PORT } = process.env;
+
+/**
+ * initializes and runs the nestjs app
+ */
+
+async function bootstrap(): Promise<void> {
```

Comment:
Might be worth checking out [this](https://docs.nestjs.com/techniques/hot-reload) (if you haven't already)

Reply:
I think we also discussed this offline, but ever time I tried to create a double-hot-reloading situation (hot-reloading server code that hot-reloads client code) I always ran into a wall of errors. I'm hoping the current setup with `nodemon`/`ts-node` will work out for us, but we can revisit if the recompile time becomes an issue.